### ISSUE TITLE
Tritonserver shared library missing on windows

### ISSUE THREAD

**Issue by SiBensberg** [triton-inference-server/server#6182]
**Description**
I built the tritonserver with docker image on windows. The server executes successfully but the shared library for the in-process API is missing. There are only the bin and include directories in the final docker image. Why are they missing? I used the default build instructions from the readme.

**Triton Information**
What version of Triton are you using?
r23.04

Are you using the Triton container or did you build it yourself?
Build myself with docker.

**Expected behavior**
Shared libraries of tritonserver are built.

**Comment by nv-kmcgill53** [comment]
Hi @SiBensberg, I would like some clarification on your use case. When you say "the shared library for the in-process API is missing," are you running on WSL or are you running on Windows? Are you able to provide steps to reproduce the build issue you are seeing? Thanks!

**Comment by SiBensberg** [comment]
Hi, @nv-kmcgill53 
I wanted to try out the in-process API as described [here](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/customization_guide/inference_protocols.html). But the described "/opt/tritonserver/lib" directory is missing in my build while headers, dll's and backends are all there. I wanted to make shure I didnt miss a flag or something when building. 

I am on windows and the container is also.
For building I simply followed the instructions. 
First I built the win10-py3-min image with TensorRT and CUDNN.
Then I used 
```python build.py --cmake-dir=<path/to/repo>/build --build-dir=/tmp/citritonbuild --no-container-pull --image=base,win10-py3-min --enable-logging --enable-stats --enable-tracing --enable-gpu --endpoint=grpc --endpoint=http --repo-tag=common:r23.04 --repo-tag=core:r23.04 --repo-tag=backend:r23.04 --repo-tag=thirdparty:r23.04 --backend=ensemble --backend=tensorrt:r23.04 --backend=onnxruntime:r23.04 --backend=openvino:r23.04```
to build the server.
I then extracted the directory from the docker to run the server with GPU support. 
The server works fine but the lib dir is missing.
